### ISSUE TITLE
Settings Menu Light/Dark Mode Switch

### ISSUE THREAD

**Issue by VasilisPat** [OrangeJedi/Aerial#37]
Would't be nice for the settings app to automatically change color mode based on the system selected mode? It's more of a final touch-up than a functional improvement, so there is no need to prioritize if approved. It would be nice to be embedded in final testing before stable release.

**Comment by OrangeJedi** [comment]
Yeah. I should be possible to adjust the color scheme to the system light/dark mode. I'll put it on the list and hopefully get around to it.

**Comment by pparado** [comment]
@OrangeJedi : Any updates on this one yet? :)

**Comment by OrangeJedi** [comment]
Yes! I have a few ideas for improving the settings menu that I want to focus on in v1.3.0. I know that's not the next release, but I don't want to take much longer on releasing the features I have already built for v1.2.0.

**Comment by pparado** [comment]
Thanks! Enjoying it for what it is at this time! 

Q: 
1. Why does it go to a blank/black screen after a while? almost feels like there is no video to play. Auto blank sceen is disabled.
2. Total files downloaded is 35.3GB as of today. Does that sound right to you?
3. A restart of the app is always necessary when ANY settings is changed? The 'refresh' don't do it?

**Comment by OrangeJedi** [comment]
> Thanks! Enjoying it for what it is at this time!
> 
> Q:
> 
>     1. Why does it go to a blank/black screen after a while? almost feels like there is no video to play. Auto blank sceen is disabled.
> 
>     2. Total files downloaded is 35.3GB as of today. Does that sound right to you?
> 
>     3. A restart of the app is always necessary when ANY settings is changed? The 'refresh' don't do it?

No problem! Here are some answers:

1. Is your computer (Windows settings) set to sleep or blank the screen automatically? Aerial won't keep your PC from sleeping on its own.
2. Yes, assuming you have every video downloaded. You can shrink this by limiting the videos that Aerial will play in the videos tab and using the `delete unselected videos` setting.
3. Not quite sure what you are referring to here. Most settings do not require the app to restart to take effect. A few, like the option to have the app run in the tray, do. The refresh button simply double checks all the stored values and initializes them to the default values if it can't read them. If there is a setting that you notice isn't updating correctly, you can create a `bug` issue.

Hopefully that helps!
